colors/palenight.kak

```diff
@@ -0,0 +1,96 @@
+# palenight theme
+
+# This was ported from https://github.com/drewtempelmeyer/palenight.vim
```

Comment:
How was this ported ? I am worried this means that this palenight.kak file is now a derivative work, and hence cannot be published inside Kakoune codebase as a UNLICENSED file.

Reply:
Hmm, I didn't think about that.  My exact process:

* I copied desertex.kak to palenight.kak
* Extracted the color numbers from https://github.com/drewtempelmeyer/palenight.vim/blob/master/autoload/palenight.vim using  Kakoune multiple cursors, and pasted them into palenight.kak as a comment.
* Updated various face commands to use rgb:xxxx, usually retyping the color number from the comment.  Read various bits of palenight.kak for reference, but not copied.
* Decided that was dumb, and reformatted the comment into shell variables.

So, everything aside from the colors would be a derivative of `desertex.kak`.  If you think that's too much of a risk, I'll split this out to another repo and make it MIT.  Or we could just make palenight.kak MIT with a copyright header and attribution in comments.

Reply:
That's how most of the builtin colorschemes were written I reckon, mines definitely so.

Reply:
I am not a lawyer, but seems good enough to me, if you started with palenight.vim and progressively massaged it into palenight.kak I would have been a bit more worried, but here seems you used palenight.vim to read the color values, which as far as I know is not something you can claim copyright on.